### PR TITLE
Register get_cpu_capability for jit

### PR DESCRIPTION
**Pull request by vfdev-5**
Description:

Context: In torchvision we ensure that functional ops are torchscriptable. Recently exposed `torch.backends.cpu.get_cpu_capability()` in https://github.com/pytorch/pytorch/pull/100164 is failing in torchvision CI
```
RuntimeError:
Python builtin <built-in function _get_cpu_capability> is currently not supported in Torchscript:
  File "/usr/local/lib/python3.10/dist-packages/torch/backends/cpu/__init__.py", line 17
    - "AVX512"
    """
    return torch._C._get_cpu_capability()
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
```
Ref: https://github.com/pytorch/vision/pull/7557

In this PR, `torch._C._get_cpu_capability()` is explicitly registered for JIT and tested.

cc @NicolasHug

### CODE REVIEW

File: test/test_torch.py [hunk 1]
```diff
@@ -7445,8 +7445,13 @@ def test_parallel_info(self):
         torch.__config__.parallel_info()
 
     def test_get_cpu_capability(self):
+        # This method is primarily exposed for torchvision's resize
         torch.backends.cpu.get_cpu_capability()
 
+        # We have to ensure that method is torchscriptable as torchvision's resize
+        # should be torchscriptable
+        torch.jit.script(torch.backends.cpu.get_cpu_capability)
```
Comment (vfdev-5) [inline test/test_torch.py:7453]: Please let me know where to put this test if it is inappropriate to put it here

**Review by albanD — APPROVED** [review]
Note that TorchScript is not really maintained anymore and we recommend users stop using it.
I'm happy to accept this one as it is pretty benign.

File: torch/csrc/jit/runtime/register_special_ops.cpp [hunk 2]
```diff
@@ -242,21 +242,21 @@ void createTensorFromList(Stack& stack) {
...
-          auto result = at::split_with_sizes(
-              (std::move(peek(stack, 0, 3))).toTensor(),
-              (std::move(peek(stack, 1, 3))).toDimVector(),
-              (std::move(peek(stack, 2, 3))).toInt());
-          drop(stack, 3);
-          pack(stack, std::move(result));
-        },
-        aliasAnalysisFromSchema()),
+RegisterOperators reg(
+    {OperatorGenerator(
```
[the remark this answers is not in the document]
Comment (vfdev-5) [reply torch/csrc/jit/runtime/register_special_ops.cpp]: Thanks for catching that! Reverted this.

**Comment by vfdev-5** [comment]
> Note that TorchScript is not really maintained anymore and we recommend users stop using it. I'm happy to accept this one as it is pretty benign.

Yes, we know that but we keep it in torchvision due to BC

**Pull request closed without merging**